include/itkSpectra1DAveragingImageFilter.hxx

```diff
@@ -159,7 +181,7 @@ Spectra1DAveragingImageFilter<TInputImage, TOutputImage>::GenerateData()
           ind1[0] = iIt.GetIndex()[0]; // set the index along the depth dimension
 
           OutputPixelType p = output->GetPixel(ind1);
-          p = p + iIt.Get();
+          p += iIt.Get();
```

Comment:
I don't remember any more why I chose operator`+` instead of operator`+=`.

Reply:
No problem, we would not have identified this ITK issue otherwise 😃